### PR TITLE
py: `ak` module can be missing, add tests

### PR DESCRIPTION
**Pull request by itayd**

### CODE REVIEW

File: runtimes/pythonrt/pythonrt.go [hunk 1]
```diff
@@ -238,7 +236,7 @@ func (py *pySvc) handleLog(msg Message) error {
 func (py *pySvc) loadSyscall(values map[string]sdktypes.Value) error {
 	ak, ok := values["ak"]
 	if !ok {
-		return fmt.Errorf("`ak` not found")
+		return nil
```
Comment (tebeka) [inline runtimes/pythonrt/pythonrt.go:262]: Log a warning

File: runtimes/pythonrt/py.go [hunk 2]
```diff
@@ -331,6 +333,8 @@ func pyExports(ctx context.Context, pyExe string, fsys fs.FS) ([]Export, error)
 		return nil, fmt.Errorf("inspect: %w.\nPython output: %s", err, buf.String())
 	}
 
+	sdklogger.Warn(ctx, "inspect output", zap.String("output", buf.String()))
```
Comment (tebeka) [inline runtimes/pythonrt/py.go]: I prefer to pass the logger from pythonrt

**Review by tebeka — APPROVED** [review]
Two comment,  not blocking merge.

File: runtimes/pythonrt/py.go (hunk 2 above)
Comment (itayd) [reply runtimes/pythonrt/py.go]: no idea why i even added this warn

File: runtimes/pythonrt/pythonrt.go (hunk 1 above)
Comment (itayd) [reply runtimes/pythonrt/pythonrt.go:262]: i don't think this needs a warning, it's completely legit not to have it if not running in a session.

**Pull request merged**